content/hardware/03.nano/boards/nano-rp2040-connect/features.md

```diff
@@ -6,63 +6,62 @@ The feature packed **Arduino Nano RP2040 Connect** brings the new **Raspberry Pi
 
-  <FeatureLink title="Datasheet" url="https://content.u-blox.com/sites/default/files/NINA-W10_ProductSummary_UBX-17051775.pdf" download blank/>
+  <FeatureLink variant="primary" title="Documentation" url="/tutorials/nano-rp2040-connect/rp2040-ap-web-server-rgb"/>
+  <FeatureLink variant="secondary" title="Library" url="https://www.arduino.cc/reference/en/libraries/wifinina/"/>
 </Feature>
 
-<Feature title="ST LSM6DSOX 6-axis IMU" image="imu">
+<Feature title="Omnidirectional Microphone" image="microphone">
 
-  Combined accelerometer and gyroscope with a dedicated machine learning core.
+  The Nano RP2040 connect comes with the MP34DT05 microphone. It allows you to capture and analyze sound in real time and can be used to create a voice interface to control your peripherals through sound.
 
-  <FeatureLink title="Datasheet" url="https://www.st.com/resource/en/datasheet/lsm6dsox.pdf" download blank/>
+  <FeatureLink variant="primary" title="Documentation" url="/tutorials/nano-rp2040-connect/rp2040-microphone-basics"/>
+  <FeatureLink variant="secondary" title="Library" url="/learn/built-in-libraries/pdm"/>
 </Feature>
 
-<Feature title="Omnidirectional Digital Microphone" image="microphone">
+<Feature title="Bluetooth" image="bluetooth">
 
-  The MP34DT05 microphone allows to capture and analyze sound in real time and can be used to create a voice interface for your project.  
+  The board is Bluetooth® enabled allowing you to control peripheral devices via bluetooth.
 
-  <FeatureLink title="Datasheet" url="https://content.arduino.cc/assets/Nano_BLE_Sense_mp34dt05-a.pdf" download blank/>
+  <FeatureLink variant="primary" title="Documentation" url="/tutorials/nano-rp2040-connect/rp2040-ble-device-to-device"/>
+  <FeatureLink variant="secondary" title="Library" url="https://www.arduino.cc/reference/en/libraries/arduinoble/"/>
 </Feature>
 
-<Feature title="Microchip ATECC608A Cryptographic Co-processor" image="crypto-chip">
+<Feature title="6-axis IMU" image="imu">
 
-  The powerful, low-power processor that is used in all MKR Family boards.
+  Combined accelerometer and gyroscope with a dedicated machine learning core.
 
-  <FeatureLink title="Datasheet" url="https://ww1.microchip.com/downloads/en/DeviceDoc/ATECC608A-CryptoAuthentication-Device-Summary-Data-Sheet-DS40001977B.pdf" download blank/>
+  <FeatureLink variant="primary" title="Documentation" url="/tutorials/nano-rp2040-connect/rp2040-imu-advanced"/>
+  <FeatureLink variant="secondary" title="Library" url="https://github.com/stm32duino/X-NUCLEO-IKS01A3"/>
 </Feature>
 
 <Feature title="Temperature Sensor" image="temperature-sensor">
 
   The LSM6DSOX sensor also features an embedded sensor that can be accessed directly via a library.
 
-  <FeatureLink title="Datasheet" url="/tutorials/nano-rp2040-connect/rp2040-01-technical-reference#temperature"/>
+  <FeatureLink variant="primary" title="Documentation" url="/tutorials/nano-rp2040-connect/rp2040-01-technical-reference"/>
```

Comment:
```suggestion
  <FeatureLink variant="primary" title="Documentation" url="/tutorials/nano-rp2040-connect/rp2040-01-technical-reference#temperature"/>
```

In this case, it is important to point to the exact section as the cheat sheet is very long (see the `#temperature`) added to end of URL.